SketchUp/Geom/LatLong.rb

```diff
@@ -15,15 +15,30 @@ class Geom::LatLong
   #   ll = [40.01700, 105.28300]
   #   latlong = Geom::LatLong.new(ll)
   #   if (latlong)
-  #     UI.messagebox latlong
+  #     UI.messagebox(latlong)
   #   else
-  #     UI.messagebox "Failure"
+  #     UI.messagebox("Failure")
   #   end
   #
-  # @param latlong
-  #   A latitude and longitude coordinate set as an Array.
+  # @overload initialize
   #
-  # @return latlong - a LatLong object
+  #   @return [Geom::LatLong]
+  #
+  # @overload initialize(latlong)
+  #
```

Comment:
Removed the empty lines.
I had adopted that as it was already in Point3d/Vector3d and other files. I suppose it should be changed there as well?